benchmarks/benchmarks.rb

```diff
@@ -115,7 +121,7 @@ def positional_splat_keyword_and_double_splat_args(a, *args, b:, **kwargs)
     # retrieval time.
     instance.no_args
 
-    x.report("#{benchmark_gem.benchmark_name}: no_args") { instance.no_args }
+    x.report("#{benchmark_gem.benchmark_name}: ()") { instance.no_args }
```

Comment:
I updated the names of benchmarks to make it clearer where they go in the README.